safety_prototype/README.rst

```diff
@@ -0,0 +1,64 @@
+README
+======
+
+Overview
+--------
+
+This is a prototype to implement `error handling`_ in Python.
+
+The user only calls API functions or methods, never SDK functions or methods.
+
+Every SDK must implement every public function or method defined in the API
```

Comment:
What happens if they don't? Will we crash on initialization or will missing/bad methods just always return the "default" value?

Reply:
Good question, I am ok with crashing on initialization.